### ISSUE TITLE
Evenly distributed images requirement should not apply to custom configurations

### ISSUE THREAD

**Issue by andre15silva** [RAR27/dyn-wall-rs#11]
I edited the config file to have 11 times, the same number as the number of images present in the images directory.

When running the command `dyn-wall-rs --custom .config/wallpapers/` I get the following error message:
`Cannot schedule the rotation of 11 images evenly throughout the day (the number of images should divide evenly into 1440)`

My config file is as follows:
```
00:00
02:00
04:00
06:00
08:00
10:00
12:00
14:00
16:00
18:00
20:00
```

I am running dyn-wall-rs version 1.1.1 on Arch Linux, installed through the AUR. Thank you!

Edit: Markdown was poorly formatted

**Comment by rehanzo** [comment]
If I count correctly, there are 10 times in your config file. However, this is still an issue, as the error message was completely wrong. I'll look into this. In the meantime, add another time or remove an image from the directory, and see if it works.

**Comment by andre15silva** [comment]
~The same error message appears with the 11 times. I must have erased one by mistake when I removed the auto-generated comments.~

It appears the Markdown was poorly formatted in the comment above. Fixed it!

**Comment by rehanzo** [comment]
Fixed this issue on v1.1.2 (7048a09b26d9b0398834a07d5ee0e1b631bb92f7). You can now update through the AUR. Thanks for reporting the issue!
